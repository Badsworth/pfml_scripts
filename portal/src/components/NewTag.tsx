import React from "react";
import { useTranslation } from "../locales/i18n";

const NewTag = () => {
  const { t } = useTranslation();

  return (
    <strong className="usa-tag radius-md margin-left-1">
      {t("components.newTag")}
    </strong>
  );
};

export default NewTag;
