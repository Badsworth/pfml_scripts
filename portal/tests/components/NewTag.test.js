import NewTag from "../../src/components/NewTag";
import React from "react";
import { render } from "@testing-library/react";

describe("NewTag", () => {
  it("renders the component", () => {
    const { container } = render(<NewTag />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
