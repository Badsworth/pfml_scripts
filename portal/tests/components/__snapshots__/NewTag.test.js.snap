// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewTag renders the component 1`] = `
<strong
  class="usa-tag radius-md margin-left-1"
>
  New
</strong>
`;
